app/scripts/lib/buy-url.js

```diff
@@ -57,16 +57,16 @@ const createWyrePurchaseUrl = async (walletAddress, chainId) => {
  *
  * @param {string} walletAddress - Ethereum destination address
  * @param {string} chainId - Current chain ID
+ * @param {string|undefined} symbol - Token symbol to buy
  * @returns String
  */
-const createTransakUrl = (walletAddress, chainId) => {
+const createTransakUrl = (walletAddress, chainId, symbol) => {
   const { transakCurrencies, network } = BUYABLE_CHAINS_MAP[chainId];
 
   const queryParams = new URLSearchParams({
     apiKey: TRANSAK_API_KEY,
     hostURL: 'https://metamask.io',
-    cryptoCurrencyList: transakCurrencies.join(','),
```

Comment:
any idea why we dropped `cryptoCurrencyList`

Reply:
Same response as the Coinbase case. Its a _show list_ vs _preselect_ the asset option

Reply:
Ok sure, but I was thinking if we need to maintain `cryptoCurrencyList` prop as it is being used to purchase assets on the network level, but we can limit that list into an array of a single asset if the purchase is intended on token level, something like this: 
`cryptoCurrencyList: symbol ? [symbol] : transakCurrencies.join(',')`

Reply:
im probably missing the reason to maintain `cryptoCurrencyList` since even at a network level, the UX is actually the same. you end up landing on ETH as preselected since its the first of the list. since we are specifying the network, it doesn't actually make an impact, because the list is prefiltered anyways by the network. `transakCurrencies` list is still worth keeping, since using that list, we can reference which tokens are "buyable" on said provider

Reply:
after discussing this we agree to get rid of `cryptoCurrencyList` since it would be redundant in this case

Reply:
yes I agree, omitting `cryptoCurrencyList` would even increase tokens coverage on specific chain and offload to maintain or provide the list from our side. make sense.